CRM/Export/BAO/Export.php

```diff
@@ -2063,8 +2066,6 @@ private static function fetchRelationshipDetails($relDAO, $value, $field, &$row)
       else {
         $fieldValue = '';
       }
-      $field = $field . '_';
```

Comment:
You're moving this line out of the loop which means now it will only happen once instead of happening multiple times. Was that your intention?